TS.fsx

```diff
@@ -243,6 +243,11 @@ let EmitEnums () =
     let emitEnum (e: Browser.Enum) = Pt.printl "declare var %s: string;" e.Name
     browser.Enums |> Array.iter emitEnum
 
+let EmitEventHandlerThis = function
```

Comment:
We have a function `GetGlobalPollutor` which will get the global pollutor type according to the spec. So it will be
```fs
let EmitEventHandlerThis = function
| _, "" -> "this: this"
| flavor, _ -> match GetGlobalPollutor flavor with
               | Some pollutor -> "this: " + pollutor
               | _ -> ""
```